plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,114 @@
+package stomp
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net"
+
+	"github.com/go-stomp/stomp"
+
+	"github.com/influxdata/telegraf"
+	commontls "github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+type STOMP struct {
+	Host      string          `toml:"host"`
+	Username  string          `toml:"username,omitempty"`
+	Password  string          `toml:"password,omitempty"`
+	QueueName string          `toml:"queueName"`
+	Log       telegraf.Logger `toml:"-"`
+
+	commontls.ClientConfig
+
+	conn  net.Conn
+	stomp *stomp.Conn
+
+	serialize serializers.Serializer
+}
+
+func (q *STOMP) Connect() error {
+	tlsConfig, err := q.ClientConfig.TLSConfig()
+	if err != nil {
+		return err
+	}
+	if tlsConfig != nil {
+		q.conn, err = tls.Dial("tcp", q.Host, tlsConfig)
+		if err != nil {
+			return err
+		}
+	} else {
+		q.conn, err = net.Dial("tcp", q.Host)
+		if err != nil {
+			return err
+		}
+	}
+
+	q.stomp, err = stomp.Connect(q.conn, stomp.ConnOpt.HeartBeat(0, 0), stomp.ConnOpt.Login(q.Username, q.Password))
```

Comment:
Does it make sense to let the user configure the heart-beat parameters?

Reply:
nice idea